xoops_trust_path/modules/xupdate/admin/forms/ModuleStoreFilterForm.class.php

```diff
@@ -46,7 +46,7 @@ class Xupdate_Admin_ModuleStoreFilterForm extends Xupdate_AbstractFilterForm
         MODULE_SORT_KEY_CATEGORY_ID => "category_id"
     );
 
-    public function Xupdate_Admin_ModuleStoreFilterForm()
```

Comment:
@mambax7 Please keep it as a backward compatible function taking into account what is called elsewhere.

e.g
```php
public function Xupdate_Admin_ModuleStoreFilterForm()
{
    self::___construct();
}
```

Thanks! 👍

Reply:
fixed and submitted.